Src/Campus.Master.API/UI/src/app/core/core.module.ts

```diff
@@ -0,0 +1,42 @@
+import { NgModule, Optional, SkipSelf } from '@angular/core';
+import { SignInService } from './services/sign-in.service';
+import { TranslateLoader, TranslateModule, TranslateStore } from '@ngx-translate/core';
+import { HTTP_INTERCEPTORS, HttpClient } from '@angular/common/http';
+import { Locales } from '@core/enum/locales.enum';
+import { LocaleService } from '@core/services';
+import { HttpLoaderFactory } from '@core/factories/locale';
+import { HttpTokenInterceptor } from '@core/http.token.interceptor';
+import { CanActivateDashboardGuard } from '@core/guards/can-activate-dashboard.guard';
+
+@NgModule({
+  providers:
+    [
+      CanActivateDashboardGuard,
+      SignInService,
+      LocaleService,
+      TranslateStore,
+      {
+        provide: HTTP_INTERCEPTORS,
+        useClass: HttpTokenInterceptor,
+        multi: true
+      }],
+  imports: [
+    TranslateModule.forChild({
```

Comment:
In Core module it's better to use `forRoot()` configuration. Here is an [example](https://github.com/ngx-translate/core#1-import-the-translatemodule)

Reply:
Replaced